css/properties/ruby-position.json

```diff
@@ -15,7 +15,7 @@
               "version_added": "12"
             },
             "edge_mobile": {
-              "version_added": true
+              "version_added": false
```

Comment:
It's not clear to me what the source is for this

Reply:
Hm. I'm not sure either, I think I actually wanted to mirror the Edge desktop version here, so this should be 12. Updating it.